openmeetings-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -55,34 +57,6 @@
 		</init-param>
 	</filter>
 
-	<!-- Start Prometheus Filter HTTP Servlet metrics
```

Comment:
@sebawagner this part was changed, could you please take a look if it will work for you? :)))